packages/grafana-ui/src/components/Drawer/_Drawer.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../node_modules/rc-drawer/assets/index.css';
+
+.drawer-content {
```

Comment:
Can you please move this styles to emotion? rc-drawer allows you to configure the `className` property which ends up on a div that wrapps the `.drawer-content` element https://github.com/react-component/drawer/blob/master/src/DrawerChild.tsx#L495

If we pass our custom class name to the RcDrawer, then we should be able to have a nested selector in Emotion styles thsat targets the content and relys on JS theme:

sth like this :
```ts
const getStyles = stylesFactory((theme) => ({
  drawer: css`
    .drawer-content {
      // styles
    }
  `
}))
```


and then in Drawer:
```tsx
return (
  <RcDrawer className={getStyles().drawer} {...} />
);
```